src/scripts.js

```diff
@@ -8,42 +8,60 @@ import SleepRepo from "./SleepRepo";
 import Activity from "./Activity";
 import ActivityRepo from "./ActivityRepo";
 
-let activityData;
-let userData;
-let userRepo;
-let activityRepo;
-let activity;
-let user;
-const uniqueUserIndex = Math.floor(Math.random() * (50 - 1 + 1)) + 1;
-
-fetch('https://fe-apps.herokuapp.com/api/v1/fitlit/1908/users/userData')
-.then(response => response.json())
-.then(data => userData = data.userData)
-.then(() => {
-userRepo = new UserRepo(userData);
-user = new User(userData[uniqueUserIndex]);
-})
-.then(() => {activityFetch(),setTimeout(all, 100)})
+//Generate random user
 
+const uniqueUserIndex = Math.floor(Math.random() * (50 - 1 + 1)) + 1;
 
-//Generate random user
-function activityFetch() {
-fetch('https://fe-apps.herokuapp.com/api/v1/fitlit/1908/activity/activityData')
-.then(response => response.json())
-.then(data => activityData = data.activityData)
-.then(() => {
-     activity = new Activity(activityData, user);
-     activityRepo = new ActivityRepo(activityData, userData);
-     date = activityData.sort((a,b) => {return new Date(b.date) - new Date(a.date) })[0].date
-     if (new Date(date) > new Date('2020/01/22')) {
-       date = '2020/01/22';
-     }
+// fetch('https://fe-apps.herokuapp.com/api/v1/fitlit/1908/users/userData')
```

Comment:
Remove commented out code.

Reply:
@ckoga This is here so you can see the two ways of doing the Fetch - sorry about that, it was not supposed to be in this PR.